src/index.tsx

```diff
@@ -0,0 +1,19 @@
+import './index.css';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import App from './components/App';
+
+ReactDOM.render(
+    <React.StrictMode>
+        <App />
+    </React.StrictMode>,
+    document.getElementById('root')
+);
+
+// Hot Module Replacement (HMR) - Remove this snippet to remove HMR.
```

Comment:
Is there a way to enable this only for a DEV environment? 
Do we want this lines in PROD ?

Reply:
Just researched that one a bit, this link is the most helpful.

https://github.com/snowpackjs/esm-hmr#esm-hmr-api-overview

Specifically:

>  You can expect your production build to strip out if (import.meta.hot) { ... } as dead code.